src/theory/tools/wp/bir_htScript.sml

```diff
@@ -1,21 +1,25 @@
 open HolKernel Parse boolLib bossLib;
 
-(* From /core: *)
+(* Theories from theory/bir: *)
 open bir_programTheory bir_typing_progTheory bir_envTheory
      bir_auxiliaryTheory bir_valuesTheory bir_expTheory
      bir_exp_immTheory bir_typing_expTheory bir_immTheory;
 
-(* From /theories: *)
+(* Theories from theory/bir-support: *)
 open bir_program_blocksTheory bir_program_terminationTheory
      bir_program_valid_stateTheory bir_exp_substitutionsTheory
      bir_bool_expTheory bir_program_env_orderTheory
      bir_program_multistep_propsTheory bir_exp_equivTheory
      bir_subprogramTheory;
 
+(* Local theories: *)
 open bir_wpTheory;
 
+(* Simplification sets from theory/bir: *)
 open HolBACoreSimps;
 
+(* This theory contains theorems for manipulation of Hoare
+ * triples not directly related to WP propagation. *)
 val _ = new_theory "bir_ht";
 
 val bir_stmtb_is_not_assume_def = Define `
```

Comment:
You forgot to rename `stmtb` to `stmtB` in this one and the next theorem.

Reply:
It's been addressed now!